packages/dotnet/ReadmeMetricsAPI6/ReadmeMetricsAPI6.csproj

```diff
@@ -9,7 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="RestSharp" Version="107.0.2" />
+    <PackageReference Include="RestSharp" Version="106.13.0" />
```

Comment:
I saw the comment in the other section, what was happening with the old version? I don't remember having any issues with the v6 library

Reply:
We were getting this issue when running the dotnet application and v6:

> Could not load type 'RestSharp.IRestResponse' from assembly 'RestSharp, Version=107.0.2

https://github.com/restsharp/RestSharp/issues/1789